src/libraries/dbLogger.ts

```diff
@@ -0,0 +1,104 @@
+import { LOG, LOG_PATH, TransactionLogTypes } from "../constants";
+import type { Query, Schema, Document } from "mongoose";
+import winston from "winston";
+
+export type TransactionLogInfo = {
+  timestamp: string;
```

Comment:
1. There are no tests written for this file. 
2. Try to get the test coverage up to 100%

Reply:
done